ros2component/ros2component/verb/standalone.py

```diff
@@ -39,13 +41,21 @@ def main(self, *, args):
         container = run_standalone_container(container_node_name=args.container_node_name)
 
         with DirectNode(args) as node:
-            load_component_into_container(
-                node=node, remote_container_node_name=args.container_node_name,
-                package_name=args.package_name, plugin_name=args.plugin_name,
-                node_name=args.node_name, node_namespace=args.node_namespace,
-                log_level=args.log_level, remap_rules=args.remap_rules,
-                parameters=args.parameters, extra_arguments=args.extra_arguments
-            )
+            try:
+                load_component_into_container(
+                    node=node, remote_container_node_name=args.container_node_name,
+                    package_name=args.package_name, plugin_name=args.plugin_name,
+                    node_name=args.node_name, node_namespace=args.node_namespace,
+                    log_level=args.log_level, remap_rules=args.remap_rules,
+                    parameters=args.parameters, extra_arguments=args.extra_arguments
+                )
+            except RuntimeError as ex:
+                # In case the component fails to load, kill the container.
+                print("{}, terminating container.".format(ex))
```

Comment:
Since the code only sends `SIGINT`s the message should mentioned "interrupting" or similar since the process is not being "terminated" (with a `SIGTERM`).

Reply:
3b9f983